flame/main.cpp

```diff
@@ -193,8 +193,13 @@ int main(int argc, char *argv[])
     long c_count = args["-c"].asLong();
 
     Protocol proto{Protocol::UDP};
-    if (args["-P"].asString() == "tcp" || args["-P"].asString() == "tcptls") {
-        proto = (args["-P"].asString() == "tcptls") ? Protocol::TCPTLS : Protocol::TCP;
+    if (args["-P"].asString() == "tcp" || args["-P"].asString() == "dot" || args["-P"].asString() == "tcptls") {
+        if (args["-P"].asString() == "dot" || args["-P"].asString() == "tcptls") {
```

Comment:
Maybe add a comment explaining that `"tcptls"` was the old way to specify `"dot"`?